Adds example user data (post deployment) scripts in bash and powershell

grdnrio opened the pull request:
This PR adds examples of user data that can be run as part of an infrastructure deployment (AWS etc) to automate the bootstrapping and first run of a node.

chef-delivery commented:
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

jamescott commented:
@delivery approve

chef-delivery commented:
Change: 74d56ffe-26ec-4236-8f0a-cc5286aa0ab2 approved by: @jamescott

chef-delivery commented:
This PR has passed 'Acceptance' and is ready to be delivered!
  Use: '@delivery deliver' when validated in acceptance.

jamescott commented:
@delivery deliver

chef-delivery commented:
Change: 74d56ffe-26ec-4236-8f0a-cc5286aa0ab2 delivered by: @jamescott

grdnrio committed:
Added User Data to install_bootstrap.rst page

My change includes User Data in Powershell and Bash to be used for unattended cloud deployment bootstrapping. This is based on demands I have faced in the field to provide assistance, particularly with Cloud Formation based infrastructure deployment.

This is the first pass - please provide feedback on suggested changes.

grdnrio committed:
Updated bootstrap install

- removed insecure SSL references
- removed client version in ps1 example
- removed validator retrieval

grdnrio committed:
Removed insecure references in ps1 bootstrap

jjasghar commented:
Great addition, we should get this is ASAP. 👍

simfish85 reviewed (commented):
👍 great addition

Commit:
minor style changes + adding note to attribute warning

grdnrio committed:
Merge pull request #1 from brewn/user-data

minor style changes + adding note to attribute warning

brewn commented:
![tenor-235227792](https://user-images.githubusercontent.com/21040805/27186302-e6cbfe4a-51b5-11e7-9cb7-97a4b0d6bc01.gif)
